### ISSUE TITLE
parse error

### ISSUE THREAD

**Issue by Winand** [flexxui/pscript#15]
I have the following dictionary. pscript fails on ñ, 'cause if i replace it with `\xf1` it compiles.
```
langs = {'de': ['De', "Deutsch"], 'en': ['En', "English"],
         'es': ['Es', "Español"], 'fr': ['Fr', "Français"],
         'ru': ['Ru', "Русский"]}
```

Exception:
`RuntimeError: Cannot do formatted string literals yet: 'es': ['Es', "Español"], 'fr': ['Fr', "Français"],`
pscript thinks that `fr` is a prefix of a string.

**Comment by almarklein** [comment]
Reproduced, I am working on a fix. It seemed that the reported column offset is one off for strings that are dict keys :(

**Comment by almarklein** [comment]
Fixed in #17. A new release is coming soon (probably this week)

**Comment by almarklein** [comment]
Thanks for the heads-up!
